### PR TITLE
Add Traffic Monitor 2.0 documentation

### PR DESCRIPTION
**Pull request by rob05c**

### CODE REVIEW

File: docs/source/admin/traffic_monitor_golang.rst [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+* CentOS 6
+* 8 vCPUs
+* 16GB RAM
+* Successful install of Traffic Ops
+* Administrative access to the Traffic Ops
+* Physical address of the site
+
+1. Enter the Traffic Monitor server into Traffic Ops
+2. Make sure the FQDN of the Traffic Monitor is resolvable in DNS.
+3. Install Traffic Monitor: ``sudo yum -y install traffic_monitor``
```
Comment (dneuman64) [inline docs/source/admin/traffic_monitor_golang.rst]: Need a step here to configure, maybe with a link to the config section.

File: docs/source/admin/traffic_monitor_golang.rst [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    Starting traffic_monitor:
+
+5. Verify Traffic Monitor is running by pointing your browser to port 80 on the Traffic Monitor host.
+
+Configuring Traffic Monitor
+===========================
+
+Configuration Overview
+----------------------
+Traffic Monitor is configured using its JSON configuration file, ``traffic_monitor.cfg``. Specify the URL, username, and password for the instance of Traffic Ops for which this Traffic Monitor is a member, and start the software.  Once started with the correct configuration, Traffic Monitor downloads its configuration from Traffic Ops and begins polling caches. Once a configurable number of polling cycles completes, health protocol state is available via RESTful JSON endpoints.
```
Comment (dneuman64) [inline docs/source/admin/traffic_monitor_golang.rst]: isn't it traffic_ops.cfg?

Comment (rob05c) [inline docs/source/admin/traffic_monitor_golang.rst]: There are both, I'll add details for both

File: docs/source/admin/traffic_monitor_golang.rst (hunk 1 above)
Comment (rob05c) [inline docs/source/admin/traffic_monitor_golang.rst]: Fixed.

File: docs/source/admin/traffic_monitor_golang.rst (hunk 2 above)
Comment (rob05c) [inline docs/source/admin/traffic_monitor_golang.rst]: Done.

**Pull request closed without merging**